src/epub/text/passing.xhtml

```diff
@@ -50,9 +50,7 @@
 					<p>Irene brought her thoughts back to the present, to the letter from Clare Kendry that she still held unopened in her hand. With a little feeling of apprehension, she very slowly cut the envelope, drew out the folded sheets, spread them, and began to read.</p>
 					<p>It was, she saw at once, what she had expected since learning from the postmark that Clare was in the city. An extravagantly phrased wish to see her again. Well, she needn’t and wouldn’t, Irene told herself, accede to that. Nor would she assist Clare to realize her foolish desire to return for a moment to that life which long ago, and of her own choice, she had left behind her.</p>
 					<p>She ran through the letter, puzzling out, as best she could, the carelessly formed words or making instinctive guesses at them.</p>
-					<blockquote epub:type="z3998:letter">
-						<p>“… For I am lonely, so lonely⁠ ⁠… cannot help longing to be with you again, as I have never longed for anything before; and I have wanted many things in my life.⁠ ⁠… You can’t know how in this pale life of mine I am all the time seeing the bright pictures of that other that I once thought I was glad to be free of.⁠ ⁠… It’s like an ache, a pain that never ceases.⁠ ⁠…” Sheets upon thin sheets of it. And ending finally with, “and it’s your fault, ’Rene dear. At least partly. For I wouldn’t now, perhaps, have this terrible, this wild desire if I hadn’t seen you that time in Chicago.⁠ ⁠…”</p>
-					</blockquote>
+					<p>“… For I am lonely, so lonely⁠ ⁠… cannot help longing to be with you again, as I have never longed for anything before; and I have wanted many things in my life.⁠ ⁠… You can’t know how in this pale life of mine I am all the time seeing the bright pictures of that other that I once thought I was glad to be free of.⁠ ⁠… It’s like an ache, a pain that never ceases.⁠ ⁠…” Sheets upon thin sheets of it. And ending finally with, “and it’s your fault, ’Rene dear. At least partly. For I wouldn’t now, perhaps, have this terrible, this wild desire if I hadn’t seen you that time in Chicago.⁠ ⁠…”</p>
```

Comment:
We do want this to be a letter though, it clearly says hse's reading a letter in the previous paragraph.

Reply:
her dialog is mixed in with non-dialog in that paragraph though

Reply:
could split it into two blockquotes?

Reply:
OK, then we should end the blockquote where the quote ends, start a p, then a new blockquote to wrap up the last sentence.